data/korath/korath missions.txt

```diff
@@ -257,3 +257,337 @@ mission "Small Scale Delivery"
+				decline
+
+			label home
+			`	You hadn't realized how tense the group was until you saw them relax as your words reach them. They seem relieved at your answer.`
+				goto title
+
+			label pay
+			`	The group relaxes quickly as you see ripple of relief and pride pass through the group.`
+
+			label title
+			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "Efreti crews have families, as children learn by working with parents. For some families, three generations were lost. We will not try to salvage the ship, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
+				accept
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti"
+		government "Kor Mereti"
+		to spawn
+			random < 30
+		personality
+			staying
+
+mission "Efreti: Distress Call 4"
+	landing
+	name "Recover the fallen of <EDC-ship>"
+	description "Return the bodies of the dead <EDC-ship> crew to <destination>."
+	source "Seleptra Nak"
+	destination "Karek Fornati"
+	cargo "Efreti deceased" 13
+	to offer
+		has "Efreti: Distress Call 3: done"
+	on offer
+		conversation
+			`You enter the <EDC-ship> wearing environment suits. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
+				to display
+					"flagship crew" > 1
+			`You enter the <EDC-ship> wearing an environment suit. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suit and perhaps impossible without it, especially with no crew helping you.`
+				to display
+					"flagship crew" = 1
+
+			`	While your engineer works on refreshing the atmosphere with outside air, the rest of the crew gathers the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
+				to display
+					"flagship crew" > 2
+			`	While your crewmate works on refreshing the atmosphere with outside air, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
+				to display
+					"flagship crew" == 2
+			`	One by one, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`
+				to display
+					"flagship crew" < 2
+
+			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`
```

Comment:
```suggestion
			`	Returning to the Korath ship, you carefully gather all the personal effects you can find in the living quarters and box them for transport. You've no way of knowing if those. in what appear to be photos. are now safely recovering or lying in your hold.`
```
Adding this makes it more interesting and also helps alleviate the sudden "there are photos" without explaining that. These are aliens, let it be a bit less friendly to direct comparisons.